modules/supported-platforms-for-openshift-clusters.adoc

```diff
@@ -7,21 +7,108 @@
 [id="supported-platforms-for-openshift-clusters_{context}"]
 = Supported platforms for {product-title} clusters
 
-In {product-title} {product-version}, you can install a cluster that uses installer-provisioned infrastructure on the following platforms:
+The following table describes which platforms are supported by the different methods available for installing {product-title} clusters:
 
-* Amazon Web Services (AWS)
-* Bare metal
-* Google Cloud Platform (GCP)
-* {ibm-cloud-name}
-* Microsoft Azure
-* Microsoft Azure Stack Hub
-* Nutanix
-* {rh-openstack-first}
-** The latest {product-title} release supports both the latest {rh-openstack} long-life release and intermediate release. For complete {rh-openstack} release compatibility, see the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
-* VMware vSphere
+// Should `none` and `external` be mentioned in this table?
 
-For these clusters, all machines, including the computer that you run the installation process on, must have direct internet access to pull images for platform containers and provide telemetry data to Red Hat.
+.Supported platforms
+[cols=".^4s,.^1,.^1,.^1,.^1,options="header"]
+|====
+|Platform|Installer-provisioned infrastructure ^[1]^|User-provisioned infrastructure ^[2]^|Agent-based Installer|Assisted Installer
 
+|{aws-first}
+|X
+|X
+|
+|
+
+|Bare metal
+|X
+|X
+|X
+|X
+
+|External
+|
+|
+|X
+|X
+
+|{gcp-first}
+|X
+|X
+|
+|
+
+|{ibm-cloud-name} Classic
+|X
+|
+|
+|
+
+|{ibm-cloud-name} Virtual Private Cloud (VPC)
+|X
+|
+|
+|
+
+|{ibm-power-name}
+|
+|X
+|
+|
+
+|{ibm-z-name} or {ibm-linuxone-name}
+|
+|X
+|
+|
+
+|{azure-first}
+|X
+|X
+|
+|
+
+|{azure-full} Stack Hub
+|X
+|X
+|
+|
+
+|None
+|
+|
+|X
+|X
+
+|Nutanix
+|X
+|
+|
```

Comment:
Note for the agent installer that Nutanix will be support in OCP versions 4.19 and later

Reply:
Sounds good! I will be documenting that in another effort, and when I do that I will make sure to add it to this table as well, thanks